### DISCUSSION TITLE
Corrupted 7-Zip File?

### DISCUSSION THREAD

**Discussion by torre-plusplus** [profi200/open_agb_firm#196] [General]
I've tried extracting the .7z file on two different operating systems (Windows 11 and Ubuntu Linux) and several unrolling programs on each. Each time I'm getting a critical error when trying to extract the .firm file. This is both on the 7/30 release and the 7/25 release. The error on Linux is "7zip Bad CRC" Followed by a lengthy hex code.

Is this just me? Didn't want to create an issue if it's just my machine.

I was going to build the file from source, but realized I was missing some an armdevkit dependency.  Will go this route if the .7z files are actually fine and it's my side that messed up.

Thanks!

**Comment by profi200** [comment]
I can unpack the archive no problem. You can try to unpack it via terminal. Maybe the program you are using is buggy.

```
$ 7z x -o./unpacked ./open_agb_firm_beta_20240730.7z

7-Zip [64] 17.05 : Copyright (c) 1999-2021 Igor Pavlov : 2017-08-28
p7zip Version 17.05 (locale=de_DE.utf8,Utf16=on,HugeFiles=on,64 bits,8 CPUs x64)

Scanning the drive for archives:
1 file, 141520 bytes (139 KiB)

Extracting archive: ./open_agb_firm_beta_20240730.7z
--
Path = ./open_agb_firm_beta_20240730.7z
Type = 7z
Physical Size = 141520
Headers Size = 342
Method = LZMA:17 PPMD ARM
Solid = +
Blocks = 3

Everything is Ok

Files: 7
Size:       279946
Compressed: 141520
```

**Reply by torre-plusplus** [reply]
Thanks for this. I ran it using 7z from the command line and everything worked perfectly. Not sure why my gui programs were having trouble.

Appreciate all your work on this project!
